packages/plugin-core/src/components/lookup/buttons.ts

```diff
@@ -31,7 +31,8 @@ export type ButtonCategory =
   | "note"
   | "split"
   | "filter"
-  | "effect";
+  | "effect"
+  | "v2"; //TODO: better category name?
```

Comment:
can we do `other`? a little more descriptive then v2 and indicates we don't have a per-defined behavior yet for this

Reply:
fixed :)